src/Features/Core/Portable/Completion/Providers/SymbolMatchPriority.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Completion.Providers
+{
+    internal class SymbolMatchPriority
+    {
+        internal static int Keyword = 100;
+        internal static int PreferType = 200;
+        internal static int PreferNamedArgument = 250;
+        internal static int PreferEventOrMethod = 300;
+        internal static int PreferFieldOrProperty = 400;
```

Comment:
Just had a thought.  If people have the "qualify with this/Me" option set, we might want to bump this/Me higher than the members that the user wants to qualify.

that what if they have fields with `t` and and type t, we will still prefer `this`...